templates/ssh_config.erb

```diff
@@ -20,6 +20,14 @@
 # list of available options, their meanings and defaults, please see the
 # ssh_config(5) man page.
 
+<% if defined?(@ssh_config_include_real) -%>
```

Comment:
There needs to be corresponding spec tests for these that show what happens when this is undef, string or array.